cdap-watchdog/src/main/java/co/cask/cdap/metrics/store/timeseries/FactCodec.java

```diff
@@ -68,9 +69,9 @@ public FactCodec(EntityTable entityTable, int resolution, int rollTimebaseInterv
    * @param ts timestamp
    * @return row key
    */
-  public byte[] createStartRowKey(List<TagValue> tagValues, String measureName, long ts) {
+  public byte[] createStartRowKey(List<TagValue> tagValues, String measureName, long ts, boolean isSearch) {
```

Comment:
I'd call the param "anyAggGroup" - because this is what it means: you construct a key for a scan that is scanning within agg group defined with tagValues or within any agg group. "isSearch" is too ambiguous.